app/models/spree/tracker.rb

```diff
@@ -14,6 +16,23 @@ def self.current(store = nil)
       Spree::Tracker.where(active: true).joins(:store).where(
         "spree_stores.code = ? OR spree_stores.url LIKE ?",
         store, "%#{store}%"
+      )
+    end
+  end
+
+  def self.by_type(store = nil, type = nil)
```

Comment:
`type` can be nil? Also, I think we should use keyword arguments here to avoid passing the `store` parameter and use the default store if not passed.